order.go

```diff
@@ -69,7 +69,7 @@ type Order struct {
 	OrderNumber           int              `json:"order_number"`
 	Note                  string           `json:"note"`
 	Test                  bool             `json:"test"`
-	BrowserIp             string           `json:"browser_ip"`
+	BrowserIP             string           `json:"browser_ip"`
```

Comment:
This would be a breaking change

Reply:
We maybe want to just leave this value

Reply:
It's my bad that I didn't initially name this according to convention, but now that it's there, I would prefer not making a breaking change for a stylistic detail.

Reply:
No problem. I'll switch it back